From 262b832c3f76afc75ecc3c76005b5b7aefd09350 Mon Sep 17 00:00:00 2001
From: "Huang, Tao" <huangtao@rock-chips.com>
Date: Sat, 13 Dec 2014 12:55:21 +0800
Subject: [PATCH] arm64: rockchip: fix tb_8846 boot

uart_dbg status default disabled
disabe wireless-wlan
bootargs remove vmalloc=512M console=ttyS2
remove memory node

Signed-off-by: Huang, Tao <huangtao@rock-chips.com>
---
 arch/arm64/boot/dts/rk3368-tb_8846.dts | 13 ++++++-------
 arch/arm64/boot/dts/rk3368.dtsi        |  2 +-
 2 files changed, 7 insertions(+), 8 deletions(-)

diff --git a/arch/arm64/boot/dts/rk3368-tb_8846.dts b/arch/arm64/boot/dts/rk3368-tb_8846.dts
index 70dcb148aff0..c6551899878f 100644
--- a/arch/arm64/boot/dts/rk3368-tb_8846.dts
+++ b/arch/arm64/boot/dts/rk3368-tb_8846.dts
@@ -7,12 +7,7 @@
 
 / {
 	chosen {
-		bootargs = "vmalloc=512M console=ttyS2 earlyprintk=uart8250-32bit,0xff690000 clk_ignore_unused";
-	};
-
-	memory@00000000 {
-		device_type = "memory";
-		reg = <0x0 0x00000000 0x0 0x80000000>;
+		bootargs = "earlyprintk=uart8250-32bit,0xff690000 clk_ignore_unused";
 	};
 
 	wireless-wlan {
@@ -43,7 +38,7 @@
 		WIFI,host_wake_irq = <&gpio3 GPIO_A6 GPIO_ACTIVE_HIGH>;
 		//WIFI,reset_gpio = <&gpio0 GPIO_A2 GPIO_ACTIVE_LOW>;
 
-		status = "okay";
+		status = "disabled";
 	};
 
 	wireless-bluetooth {
@@ -381,6 +376,10 @@
 	*/
 };
 
+&uart_dbg {
+	status = "okay";
+};
+
 &uart_bt {
 	status = "okay";
 	dma-names = "!tx", "!rx";
diff --git a/arch/arm64/boot/dts/rk3368.dtsi b/arch/arm64/boot/dts/rk3368.dtsi
index a7a37ba4663b..de0fa254c701 100644
--- a/arch/arm64/boot/dts/rk3368.dtsi
+++ b/arch/arm64/boot/dts/rk3368.dtsi
@@ -406,7 +406,7 @@
 		//#dma-cells = <2>;
 		pinctrl-names = "default";
 		pinctrl-0 = <&uart2_xfer>;
-		//status = "disabled";
+		status = "disabled";
 	};
 
 	uart_gps: serial@ff1b0000 {
-- 
2.35.3

